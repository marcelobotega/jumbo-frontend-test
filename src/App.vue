<template>
  <div class="app-wrapper">
    <MainNavbar />
    <div class="app-main-content xl:container xl:mx-auto px-3 py-6">
      <router-view />
    </div>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator";
import MainNavbar from "@/components/main-navbar/MainNavbar.vue";

@Component({
  components: {
    MainNavbar,
  },
})
export default class App extends Vue {}
</script>
