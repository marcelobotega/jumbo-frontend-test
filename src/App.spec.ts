import { shallowMount } from "@vue/test-utils";
import App from "@/App.vue";

describe("App.vue", () => {
  it("should create", () => {
    const wrapper = shallowMount(App, {
      stubs: ["router-link", "router-view"],
    });
    expect(wrapper).toMatchSnapshot();
  });
});
