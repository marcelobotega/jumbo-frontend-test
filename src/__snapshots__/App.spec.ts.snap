// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue should create 1`] = `
<div class="app-wrapper">
  <mainnavbar-stub></mainnavbar-stub>
  <div class="app-main-content xl:container xl:mx-auto px-3 py-6">
    <router-view-stub></router-view-stub>
  </div>
</div>
`;
